lms/djangoapps/certificates/__init__.py

```diff
@@ -1,3 +0,0 @@
-""" Certificates app """
-# this is here to support registering the signals in signals.py
-from . import signals
```

Comment:
@rlucioni I meant to switch it over to the newer apps.py style (but forgot) -- can leave if you think __init__ is preferable.